tests/e2e/kserve_test.go

```diff
@@ -59,7 +58,6 @@ func kserveTestSuite(t *testing.T) {
 	t.Run("Validate serving transition to Unmanaged", componentCtx.ValidateServingTransitionToUnmanaged)
 	t.Run("Validate serving transition to Removed", componentCtx.ValidateServingTransitionToRemoved)
 	t.Run("Validate component disabled", componentCtx.ValidateComponentDisabled)
-	// t.Run("Validate component releases", componentCtx.ValidateComponentReleases)
```

Comment:
@ugiordan is this a test that was expected to be enabled here at some point, or has it become irrelevant?

Reply:
@grdryn Yes, I enabled it https://github.com/opendatahub-io/opendatahub-operator/pull/1738/files#diff-886ea9b04538f455209581c7bc607ad3e146cb53122a0984be6576ba4d48462a